log-insights/setup/self-managed/_01_standard_instructions.mdx

```diff
@@ -0,0 +1,4 @@
+Note that both individual files and directories are supported for `db_log_location`.
+
+If the `db_log_location` was discovered by the collector, no additional setup is
+required and you can proceed to the next step.
```

Comment:
maybe:

```
If the `db_log_location` was discovered by the collector, you can note down the location and proceed to the next step - no log location changes are required.
```

Reply:
Also, I'm wondering whether we should talk about the `pganalyze` user needing access to the file/folder here.

We've fixed this in the general setup by adding the `pganalyze` user to the `adm` group (so it can read log files in `/var/log`), but if the log directory is in a customized location, we may not be able to read it, and the customer would have to follow parts of the directions we have right now only under "in data directory". Specifically, adding the pganalyze user to the postgres group, and adjusting `log_file_mode` may also be required here.

Reply:
Hmm, good point. The location is still discovered successfully in that case because we don't actually test permissions there, right? Could we do that?

Reply:
We could test permissions, but it would require some extra logic (either manually check the file's owner and parent directories, or do a priviledge drop test)

Reply:
As discussed, for now, we'll add some checks to the "happy" path here to verify permissions.